store/sql_user_store.go

```diff
@@ -437,13 +439,13 @@ func (s SqlUserStore) GetEtagForDirectProfiles(userId string) StoreChannel {
 	return storeChannel
 }
 
-func (s SqlUserStore) GetEtagForAllProfiles() StoreChannel {
+func (s SqlUserStore) GetEtagForAllProfiles(limit int, offset int) StoreChannel {
```

Comment:
Hmm....I wonder if we should just modify this to be way more conservative or remove it.  For example what happens when I delete a user and the whole list shifts by 1 on the boundary?  then some pages will still have the same etag and never get updated.

Since profile changes are somewhat infrequent.  Maybe just `SELECT MAX(UpdateAt) FROM` for all users vs adding the paging stuff.